<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>76</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>76</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*.QWidget {  
border-top-left-radius:15px;
border-top-right-radius:15px;
}*/
#widget_top
{
background-color: rgb(36, 61, 91);

}
#widget_top_logo
{
color: rgb(122, 175, 227);
border-image: url(:/ptr/logo.png);
}
#widget_top_title
{
font: 18pt &quot;楷体&quot;;
}
#widget_top_title_English
{
font: 11pt &quot;楷体&quot;;
}
#widget_top_time
{
font: 20pt &quot;Digital-7 Mono&quot;;
}
#widget_top_year
{
font: 9pt &quot;楷体&quot;;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>79</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
color: rgb(122, 175, 227);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_widget_top">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="widget_top_logo">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_Title">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="widget_top_title">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>水产养殖自动控制系统</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="widget_top_title_English">
            <property name="minimumSize">
             <size>
              <width>370</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>370</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Automatic control system for aquaculture</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="TH_widget">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_top_function" native="true">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
color: rgb(122, 175, 227);
font: 10pt &quot;楷体&quot;;
border-radius:6px;
}
QToolButton:hover
{
background-color: rgb(19, 48, 80);
background-color: rgb(3, 57, 103);
font: 10pt &quot;楷体&quot;;
border-radius:6px;
border:2px solid rgb(13,39,67);
}</string>
          </property>
          <widget class="QToolButton" name="btn_Real_time">
           <property name="geometry">
            <rect>
             <x>29</x>
             <y>5</y>
             <width>76</width>
             <height>65</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton
{
background-color: rgb(19, 48, 80);
background-color: rgb(3, 57, 103);
font: 10pt &quot;楷体&quot;;
border-radius:6px;
border:2px solid rgb(13,39,67);
}</string>
           </property>
           <property name="text">
            <string>实时数据</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/ptr/Real_time.png</normaloff>:/ptr/Real_time.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QToolButton" name="btn_Record_query">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>5</y>
             <width>76</width>
             <height>65</height>
            </rect>
           </property>
           <property name="text">
            <string>记录查询</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/ptr/Record_query.png</normaloff>:/ptr/Record_query.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QToolButton" name="btn_Control">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>5</y>
             <width>76</width>
             <height>65</height>
            </rect>
           </property>
           <property name="text">
            <string>控制管理</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/ptr/control.png</normaloff>:/ptr/control.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QToolButton" name="btn_Fault_detect">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>5</y>
             <width>76</width>
             <height>65</height>
            </rect>
           </property>
           <property name="text">
            <string>故障检测</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/ptr/Fault_detect.png</normaloff>:/ptr/Fault_detect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_top_right" native="true">
          <property name="maximumSize">
           <size>
            <width>123</width>
            <height>79</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_top_Menu_widget" native="true">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>123</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
border-radius:3px;
}
QPushButton:hover
{
background-color: rgb(215, 215, 215);
border-radius:3px;
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>1</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="TH_Min">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="widget_top_btn_Min">
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/ptr/Min1.png</normaloff>:/ptr/Min1.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>14</width>
                  <height>14</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="TH_Max">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="widget_top_btn_Max">
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/ptr/Max1.png</normaloff>:/ptr/Max1.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>14</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="TH_Close">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="widget_top_btn_Close">
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/ptr/Close1.png</normaloff>:/ptr/Close1.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>14</width>
                  <height>14</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="TH_Close_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="widget_top_time">
             <property name="minimumSize">
              <size>
               <width>123</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>123</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Digital-7 Mono</family>
               <pointsize>20</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>14:26:45</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="widget_top_year">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>2023年11月1日</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
